cmd/minishift/cmd/root.go

```diff
@@ -71,6 +73,10 @@ var RootCmd = &cobra.Command{
 
 		ensureConfigFileExists(constants.ConfigFile)
 
+		// Initialize machine data
+		jsonDataPath := filepath.Join(constants.Minipath, "machines", constants.MachineName, "instance-data.json")
```

Comment:
👍 , good to initialize this here. Also good to create the full path here. Keeps the rest of the code clean from constants usage.